azure_arc_k8s_jumpstart/aksengine/arc_k8s_aksengine.azcli

```diff
@@ -0,0 +1,432 @@
+########################################################
+# Script to deploy different features of Arc-enabled
+#    k8s clusters, for demo or training purposes.
+#
+# Jose Moreno
+# March 2021
+########################################################
+
+
+#############################
+# Deploy aks-engine cluster #
+#############################
+
+# Variables
+k8s_rg=aksengine
+location=westeurope
+arc_rg=k8sarc
+arc_name=myaksengine
+
+# Create RG for AKS engine cluster
+az group create -n $k8s_rg -l $location
+
+# Install AKS engine
+# Go here: https://github.com/Azure/aks-engine/releases/latest
+# Example with v0.61.0:
+aksengine_exec=$(which aks-engine)
+if [[ -n "$aksengine_exec" ]]
+then
+    echo "aks-engine executable found in ${aksengine_exec}"
+else
+    echo "Downloading and installing aks-engine executable..."
+    aksengine_tmp=/tmp/aksengine.tar.gz
+    wget https://github.com/Azure/aks-engine/releases/download/v0.61.0/aks-engine-v0.61.0-linux-amd64.tar.gz -O $aksengine_tmp
+    tar xfvz $aksengine_tmp -C /tmp/
+    sudo cp /tmp/aks-engine-v0.61.0-linux-amd64/aks-engine /usr/local/bin
+fi
+
+# Retrieve Service Principal form your AKV, required for AKS engine
```

Comment:
All of the scenarios use a service principal as part of the prequisites. Although KV integration is better. For consistency I would keep it in the prequisites and leave KV aside.

Reply:
Really? Hardcoding SP secrets in scripts? I feel bad just thinking about that

Reply:
as mentioned going to discuss this next week on team meeting